backend/apps/matching/models/user.py

```diff
@@ -35,6 +38,25 @@ def new(email, pwd=None, **kwargs):
             user.set_password(pwd)
         else:
             user.set_password(email)
+
+        group_is_a = Group.objects.get(name="is_a")
+        group_is_b = Group.objects.get(name="is_b")
+        group_can_view_a = Group.objects.get(name="can_view_a")
+        group_can_view_b = Group.objects.get(name="can_view_b")
+
+        if kwargs["is_A"]:
+            user.groups.add(group_is_a)
```

Comment:
The `is_A` and `is_B` groups are not necessary for this PR, but we never fixed this before, so I added it here.